templates/500.html

```diff
@@ -40,9 +45,33 @@
             {% include 'includes/gtm-no-script.html' %}
         {% endif %}
 
-        <h1 class="tna-heading-xl">Internal server error</h1>
+        {% include 'includes/header.html' %}
+
+        <main id="maincontent">
+            <div class="generic-intro">
+                <div class="tna-container tna-container--centred">
+                    <div class="tna-column tna-column--full tna-column--width-1-2 tna-column--full-small tna-column--full-tiny tna-!--margin-vertical-l">
```

Comment:
Actually, is there meant to be a `tna-column--full` and a `tna-column--width-1-2`?